src/wwwroot/styles/views/_settings.scss

```diff
@@ -620,6 +620,67 @@
   }
 }
 
+.plan--box-container {
+  .arrow--box-up {
+    position: relative;
+    background: $suit-font-color-bright;
+    border: 1px solid $suit-accent-color;
+    color: $suit-main-color;
+    top: 10px;
+    padding: 35px;
+    &:after{
+      bottom: 100%;
+      left: 24%;
+      border: solid transparent;
+      content: " ";
+      position: absolute;
+      border-bottom-color: $suit-font-color-bright;
+      border-width: 8px;
+      margin-left: -8px;
+    }
+    &:before{
+      bottom: 100%;
+      left: 24%;
+      border: solid transparent;
+      content: " ";
+      position: absolute;
+      border-bottom-color: $suit-accent-color;
+      border-width: 9px;
+      margin-left: -9px;
+    }
+    ol{
```

Comment:
ol is for ordered lists, im not sure if this is the case, maybe UL is more common to use in this cases, just consider it

Reply:
Is an orderer list based on the mocks.